controller/predicates/is_layer_requested.js

```diff
@@ -0,0 +1,19 @@
+const _ = require('lodash');
```

Comment:
would it make sense for this predicate to look like `has_results_at_layer`? with functions for `any` and `all`? i'd imagine we would eventually want to use it that way and I kinda like the consistency across these where possible. thoughts?